forderud/IntelliMouseDriver
# Draft: Update driver to Logitech LX3 Optical Mouse and mouse mirroring

PR updated after merging #49.

Tweak driver so that it instead loads for Logitech LX3 Optical Mouse (`VID_046D&PID_C044`).

## How to test
* Update the driver HW IDs to match your mouse model (like what's done in this PR).
* Build the sources and run `INSTALL_MouseMirror.bat` to install the driver.
* Run the `MouseMirror.ps1` script to enable mirroring of mouse movement.
* Observe that mouse movements are mirrored and upside down.
* Run `UNINSTALL_MouseMirror.bat` to clean up. Afterwards, reconnect your mouse.

## Mouse HW IDs
![image](https://github.com/forderud/IntelliMouseDriver/assets/2671400/7f7aff60-2567-4644-bcdf-c11ec5c03d48)
